src/app/service/coordinape/CoordinapeSendForm.ts

```diff
@@ -11,9 +12,9 @@ export default async (member: GuildMember, ctx?: CommandContext): Promise<any> =
 	}
 
 	for (const role of member.roles.cache.values()) {
-		if (role.name === 'Level 1') {
+		if (role.id === roleIds.level1) {
```

Comment:
Typically roles are independent of one another in discord but because at BanklessDAO you can't really have level 2 without level 1, loop within a loop should be fine

Reply:
yep i confirmed it would not be possible to have lvl2 without lvl1